nixos/modules/config/networking.nix

```diff
@@ -185,8 +172,7 @@ in
           ${cfg.extraHosts}
         '';
 
-        # /etc/host.conf: resolver configuration file
-        "host.conf".text = cfg.hostConf;
+        "host.conf".text = mkDefault "multi on\n";
```

Comment:
We should add a reference to https://github.com/nixos/nixpkgs/issues/19148 here.